packages/app-store/around/api/_getAdd.ts

```diff
@@ -0,0 +1,25 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { defaultResponder } from "@calcom/lib/server";
+
+import checkSession from "../../_utils/auth";
+import { checkInstalled, createDefaultInstallation } from "../../_utils/installation";
+import appConfig from "../config.json";
+
+export async function getHandler(req: NextApiRequest, res: NextApiResponse) {
+  const session = checkSession(req);
+  const slug = appConfig.slug;
+  const appType = appConfig.type;
+
+  await checkInstalled(slug, session.user.id);
+  await createDefaultInstallation({
+    appType,
+    userId: session.user.id,
+    slug,
+    key: {},
+  });
+
+  return res.status(200).json({ url: "/apps/installed" });
```

Comment:
The default responder already does this

```suggestion
  return { url: "/apps/installed" };
```